src/components/loginSignup/login.jsx

```diff
@@ -18,9 +18,37 @@ export default class Login extends Component {
 
   handleInput = (stateName) => (e) => {
     e.preventDefault()
+    this.setState({
+      [stateName]: e.target.value.trim(),
+      errors: {
+        username: '',
+        password: '',
+      }
+    })
+    this.handleError(stateName)
     console.log(e.target.value)
   }
 
+  handleError = (stateName) => {
+    const error = this.state.errors
+    if (stateName === "username") {
```

Comment:
IF condition not needed, make it dynamic

Reply:
dynamic? hmm it is working dynamic..